src/v2/view-builder/views/consent/EnduserEmailConsentView.js

```diff
@@ -0,0 +1,91 @@
+import { loc } from 'okta';
+import { BaseForm, BaseHeader } from '../../internals';
+import ConsentViewForm from './ConsentViewForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import HeaderBeacon from '../../components/HeaderBeacon';
+import { getIconClassNameForBeacon } from '../../utils/AuthenticatorUtil';
+import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
+
+export default BaseAuthenticatorView.extend({
+  className: 'enduser-email-consent',
+
+  Header: BaseHeader.extend({
+    HeaderBeacon: HeaderBeacon.extend({
+      getBeaconClassName: function() {
+        return getIconClassNameForBeacon(AUTHENTICATOR_KEY.EMAIL);
+      },
+    }),
+  }),
+
+  postRender() {
+    // Move buttons in DOM to match visual hierarchy to fix tab order.
+    // TODO https://oktainc.atlassian.net/browse/OKTA-350521
+    const buttonContainer = this.$el.find('.o-form-button-bar');
+    buttonContainer.find('.button-primary').appendTo(buttonContainer);
+    buttonContainer.find('.button-primary').removeClass('button-primary');
+  },
+
+  Body: ConsentViewForm.extend({
+    title() {
+      return loc('oie.consent.enduser.title', 'login');
+    },
+    save() {
+      return loc('oie.consent.enduser.accept.label', 'login');
+    },
+    cancel() {
+      return loc('oie.consent.enduser.deny.label', 'login');
+    },
+    initialize() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      if (this.model.get('browser')) {
+        this.add(`
+          <div class="enduser-email-consent--info">
+            <i class="enduser-email-consent--icon browser-icon"></i>
+            <div>${this.model.escape('browser')}</div>
+          </div>
+        `);
+      }
+      if (this.model.get('app')) {
+        this.add(`
+          <div class="enduser-email-consent--info">
+            <i class="enduser-email-consent--icon app-icon"></i>
+            <div>${this.model.escape('app')}</div>
+          </div>
+        `);
+      }
+    },
+    getUISchema() {
+      const uiSchemas = BaseForm.prototype.getUISchema.apply(this, arguments);
+      return uiSchemas.filter(uiSchema => uiSchema.name !== 'consent' );
```

Comment:
what else besides `consent` in the uiSchema?

Reply:
There is only `consent` as of now. But in future, if something else gets added, that needs to be displayed. Since `consent` is a `boolean` I had to filter it out to not show it as `checkbox`.

Reply:
ah, I see. enduser / admin consent handled this is `ion-boolean-handler.js`